code/datums/outputs.dm

```diff
@@ -0,0 +1,117 @@
+//NOTE: When adding new sounds here, check to make sure they haven't been added already via CTRL + F.
+
+/datum/outputs
+    var/text = "You hear something."
+    var/list/sounds = 'sound/misc/bangindonk.ogg' //can be either a sound path or a WEIGHTED list, put multiple for random selection between sounds
+    // var/mutable_appearance/vfx = list('icons/sound_icon.dmi',"circle", HUD_LAYER) //syntax: icon, icon_state, layer
```

Comment:
Same here

Reply:
In this case i wanted to use this, but i don't know what to use, do i port the same images from /tg/?

Reply:
My mistake i added the following line. Removing